src/Sulu/Bundle/RouteBundle/Controller/HistoryRouteController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Controller;
+
+use FOS\RestBundle\Controller\Annotations\RouteResource;
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use Hateoas\Representation\CollectionRepresentation;
+use Sulu\Bundle\RouteBundle\Entity\RouteRepositoryInterface;
+use Sulu\Bundle\RouteBundle\Model\RouteInterface;
+use Sulu\Component\Rest\Exception\EntityNotFoundException;
+use Sulu\Component\Rest\RequestParametersTrait;
+use Sulu\Component\Rest\RestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides api to handle routes.
+ *
+ * @RouteResource("history-routes")
+ */
+class HistoryRouteController extends RestController implements ClassResourceInterface
```

Comment:
Shouldn't that be part of a `RouteController`? Having a `/history-routes` endpoint doesn't feel very restful to me... Something like `/nodes/uuid/routes?history=true` would suit better IMO, however every bundle would have to include that Controller as a subresource then... Not sure about that yet, but I think that would make sense.